### ISSUE TITLE
Some ship personalities should bribe you to call off your attack

### ISSUE THREAD

**Issue by Pointedstick** [endless-sky/endless-sky#253]
You can bribe enemy ships to get them to stop attacking you. I think they should be able to do the same! This sort of goes with the "no more suicidal pirates" request. Most people don't want to die; if they see that it's going to happen, they'll try to avoid it. This would gel well with all the instances in which the storyline humanizes the pirates; many are just wayward teenagers, some are slaves, etc.

This would definitely apply to the "timid" personality trait, possibly as soon as they take some damage, and to the "plunders" trait as well, if their ship is critically damaged but not yet destroyed. Definitely not for "heroic." They'll go down fighting!

**Comment by thenerdfreak** [comment]
I agree. I was trying to demand tribute from some merchants the other day and was like "why can't I do this. This would be amazing.

**Comment by Phezzan** [comment]
I like this, but logically wouldn't the pirates just try to run away?
How would the 'nearest target' button work after you are bribed ?
We'd have to tell escort ships to avoid that pirate, but not mess up local forces...

PS: killing a lot of anything should earn you a bounty. That attracts a 'special' assassin fleet!

**Comment by JadenSpef1** [comment]
> Phezzan "PS: killing a lot of anything should earn you a bounty. **_That attracts a 'special' assassin fleet!_** 

THIS IS A AWESOME IDEA!!
